diff -u --label /nix/store/8f87spmw4pbi40cvlnh788p4zny6mibg-source/lem.asd --label \#\<buffer\ lem.asd\> /nix/store/8f87spmw4pbi40cvlnh788p4zny6mibg-source/lem.asd /tmp/buffer-content-10umy1
--- a/lem.asd
+++ b/lem.asd
@@ -130,48 +130,49 @@
                              (:file "detective")))))
 
 (defsystem "lem/extensions"
-  :depends-on (#+sbcl
-	       "lem-lsp-mode"
-	       "lem-vi-mode"
-	       #+sbcl
-	       "lem-lisp-mode"
-	       #+sbcl
-	       "lem-go-mode"
+  :depends-on (;; #+sbcl
+	       ;; "lem-lsp-mode"
+	       ;; "lem-vi-mode"
+	       ;; #+sbcl
+	       ;; "lem-lisp-mode"
+	       ;; #+sbcl
+	       ;; "lem-go-mode"
 
-	       "lem-c-mode"
-	       "lem-xml-mode"
-	       "lem-html-mode"
-	       "lem-python-mode"
-	       "lem-posix-shell-mode"
-	       "lem-markdown-mode"
-	       "lem-js-mode"
-	       "lem-json-mode"
-	       "lem-css-mode"
-	       "lem-rust-mode"
-	       "lem-paredit-mode"
-	       "lem-nim-mode"
-	       #-clasp
-	       "lem-scheme-mode"
+	       ;; "lem-c-mode"
+	       ;; "lem-xml-mode"
+	       ;; "lem-html-mode"
+	       ;; "lem-python-mode"
+	       ;; "lem-posix-shell-mode"
+	       ;; "lem-markdown-mode"
+	       ;; "lem-js-mode"
+	       ;; "lem-json-mode"
+	       ;; "lem-css-mode"
+	       ;; "lem-rust-mode"
+	       ;; "lem-paredit-mode"
+	       ;; "lem-nim-mode"
+	       ;; #-clasp
+	       ;; "lem-scheme-mode"
 
-	       "lem-patch-mode"
-	       "lem-yaml-mode"
-	       "lem-review-mode"
-	       "lem-asciidoc-mode"
-	       "lem-dart-mode"
-	       "lem-scala-mode"
-	       "lem-dot-mode"
-	       "lem-java-mode"
-	       "lem-haskell-mode"
-	       "lem-ocaml-mode"
-	       "lem-asm-mode"
-	       "lem-makefile-mode"
-	       "lem-shell-mode"
-	       "lem-sql-mode"
-	       "lem-base16-themes"
-	       #+sbcl
-	       "lem-elixir-mode"
-	       "lem-documentation-mode"
-	       "lem-elisp-mode"))
+	       ;; "lem-patch-mode"
+	       ;; "lem-yaml-mode"
+	       ;; "lem-review-mode"
+	       ;; "lem-asciidoc-mode"
+	       ;; "lem-dart-mode"
+	       ;; "lem-scala-mode"
+	       ;; "lem-dot-mode"
+	       ;; "lem-java-mode"
+	       ;; "lem-haskell-mode"
+	       ;; "lem-ocaml-mode"
+	       ;; "lem-asm-mode"
+	       ;; "lem-makefile-mode"
+	       ;; "lem-shell-mode"
+	       ;; "lem-sql-mode"
+	       ;; "lem-base16-themes"
+	       ;; #+sbcl
+	       ;; "lem-elixir-mode"
+	       ;; "lem-documentation-mode"
+	       ;; "lem-elisp-mode"
+         ))
 
 (defsystem "lem/legit"
   :serial t

Diff finished.  Tue Aug 22 13:45:41 2023
